docs/dev/libvirt-howto.md

```diff
@@ -7,6 +7,9 @@ It's expected that you will create and destroy clusters often in the course of d
 
 Before you begin, install the [build dependencies](dependencies.md).
 
+### Enable BIOS settings are enabled related with virtualization
+Make sure you have already enable `Intel VT and AMD-V virtualization hardware extensions in BIOS settings`.
```

Comment:
I'd rather not have to get down to this low a level.  How about:

    ### Enable KVM

    Make sure you have KVM enabled by checking for the device:

    ```console
    $ ls -l /dev/kvm 
    crw-rw-rw-+ 1 root kvm 10, 232 Oct 31 09:22 /dev/kvm
    ```

    If it is missing, try some of the ideas [here][kvm-install].

And then at the bottom of the file:

```markdown
[kvm-install]: http://www.linux-kvm.org/page/FAQ#Preparing_to_use_KVM
```

That covers a number of things, including [enabling virtualization in BIOS][1].

[1]: http://www.linux-kvm.org/page/FAQ#.22KVM:_disabled_by_BIOS.22_error

Reply:
sounds good to me

Reply:
sound really good!